spec/relaton_ccsds/data_fetcher_spec.rb

```diff
@@ -22,13 +23,6 @@
       expect(subject.agent.request_headers["Accept"]).to eq "application/json;odata=verbose"
     end
 
-    it "#index" do
```

Comment:
@andrew2net the test start to crash after I change index-v1 to index-v2, I'm pretty sure it's not what we should test.
Checking if `FileIO` have correct directory for output make sense, but we can do it in more integrational way. Like checking the produced output file.

Reply:
I don't mind, but now this method isn't tested even by integration tests because it reimplemented in the `TestHitCollection`

<img width="831" alt="image" src="https://github.com/user-attachments/assets/c1b7de7e-67cf-4979-8afa-21aceb0d2b77">

Also a file `index-v1.yaml` appears in the gem's root dir after each test. We should't have to remove anything by hand after running tests.